js/files.ts

```diff
@@ -123,6 +127,23 @@ export async function write(rid: number, p: Uint8Array): Promise<number> {
   return res.nbyte();
 }
 
+/** Seek a file ID to the given offset under mode given by `whence`.
+ *
+ */
+export async function seek(
+  rid: number,
+  offset: number,
+  whence: number
```

Comment:
pardon the dumb question here... why is `whence` a number and not `SeekMode`?

Reply:
Fixed, it was some leftovers from the old `Seeker` interface.